### ISSUE TITLE
ParenthesizedSentence incorrectly described in Japanese documentation

### ISSUE THREAD

**Issue by edwardwoodward** [redpen-cc/redpen#729]
The Japanese documentation states that max_length checks the number of words (単語数) in parentheses. It doesn't - it checks the number of characters. Also, the default value of max_length is 4, which is too few if it is counting characters, I think.

**Comment by takahi-i** [comment]
Hi @edwardwoodward. Thank  you very much for telling us a mistake in RedPen document.  The RedPen document is also hosted in GitHub. The following is the URL. 

https://github.com/redpen-cc/redpen-doc

The document is written in a simple right weight markup language, AsciiDoc, And therefore I thinks you can make the fix for the problem.  

 That would be appreciate it if you give us the pull request for us.

**Comment by edwardwoodward** [comment]
I don't mind doing this, but I need some sort of explanation of the procedure, preferably in plain English.
Alternatively, you just give me the file you want me to edit and I'll edit it and return it.
